cosmwasm/contracts/ibc-translator/src/contract.rs

```diff
@@ -0,0 +1,111 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+
+use anyhow::{bail, Context};
+use cosmwasm_std::{
+    to_binary, Binary, Deps, DepsMut, Empty, Env, MessageInfo, Reply, Response, StdResult,
+};
+use wormhole_bindings::tokenfactory::TokenFactoryMsg;
+
+use crate::{
+    execute::{
+        complete_transfer_and_convert, convert_and_transfer, submit_update_chain_to_channel_map,
+        TransferType,
+    },
+    msg::{ExecuteMsg, InstantiateMsg, QueryMsg, COMPLETE_TRANSFER_REPLY_ID},
+    query::query_ibc_channel,
+    reply::handle_complete_transfer_reply,
+    state::{TOKEN_BRIDGE_CONTRACT, WORMHOLE_CONTRACT},
+};
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn instantiate(
+    deps: DepsMut,
+    _env: Env,
+    info: MessageInfo,
+    msg: InstantiateMsg,
+) -> Result<Response, anyhow::Error> {
+    TOKEN_BRIDGE_CONTRACT
+        .save(deps.storage, &msg.token_bridge_contract)
+        .context("failed to save token bridge contract address to storage")?;
+
+    WORMHOLE_CONTRACT
+        .save(deps.storage, &msg.wormhole_contract)
+        .context("failed to save wormhole contract address to storage")?;
```

Comment:
We can remove the wormhole contract storage variable - I don't believe it's used anywhere.

Reply:
yes, using the custom query, we can then remove this